api/pkg/apis/v1alpha1/vendors/campaigns-vendor_test.go

```diff
@@ -122,18 +122,18 @@ func TestCampaignsOnCampaignsFailure(t *testing.T) {
 		Method: fasthttp.MethodGet,
 		Body:   data,
 		Parameters: map[string]string{
-			"__name": "campaign1",
+			"__name": "campaign1-v1",
 		},
 		Context: context.Background(),
 	})
 	assert.Equal(t, v1alpha2.InternalError, resp.State)
-	assert.Equal(t, "Not Found: entry 'campaign1' is not found in namespace default", string(resp.Body))
+	assert.Equal(t, "Not Found: entry 'campaign1-v1' is not found in namespace default", string(resp.Body))
 
 	resp = vendor.onCampaigns(v1alpha2.COARequest{
 		Method: fasthttp.MethodPost,
 		Body:   []byte("bad data"),
 		Parameters: map[string]string{
-			"__name": "campaign1",
+			"__name": "campaign1-v1",
```

Comment:
I feel confused, as a path param. Should we pass "container:version" or "container-version"?

Reply:
Same as before. For create.